site/content/how-to-monetize-apps/monetization.md

```diff
@@ -0,0 +1,92 @@
+---
+weight: 10 
+---
+
+# How to monetize apps?
+
+<!--Abstract, short intro -->
+Whether your app is already up and running or you're just starting off, the topic of **app monetization** will eventually cross your mind. If you don't know how to go about generating income from your apps, this is the place for you to start.
+
+
+<!-- Intro for newcomers -->
+## Introduction
+If you're new to LiveChat, here's how things work in here. LiveChat can be extended with various applications built by developers. All of this is done to meet the specific business needs of those who use LiveChat. You, as a developer and app author, can sell it on our [Marketplace](https://www.livechatinc.com/marketplace/). 
+
+If you'd like to know what types of applications can be built upon LiveChat Platform, check out the [Building LiveChat apps](/building-apps-for-livechat/) doc. 
+
+The place to start creating your apps with different building blocks is our [Developer Console](https://developers.livechatinc.com/console/). If your app is ready, you can continue with the monetization process. 
+
+<!-- How the process looks -->
+## Core information
+
+Here's the core info about monetization at **LiveChat**:
+
+* You can perform all the actions in one place- [Developer Console](https://developers.livechatinc.com/console/), which speeds up the whole procedure. 
```

Comment:
I'd emphasize the ease of the process even more:
> Setting up monetization is easy and done entirely in Developer Console

Reply:
although it's passive voice

Reply:
😅 true! Maybe something like this:
>You can set up monetization in Developer Console with just a few easy steps